packages/matter.js-react-native/README.md

```diff
@@ -23,3 +23,25 @@ This package uses the following react-native libraries to provide the needed fun
 ## Tests
 
 No tests available for now
+
+## React Native Configuration
+
+To use \`**matter.js**\` with React Native, you need to modify your Metro configuration to ensure that the packages are resolved correctly. You need to add a special resolver entry and merge it with your existing Metro configuration. Below is an example of how to do this:
+
+```javascript
+/* eslint-env node */
+const { mergeConfig } = require('@react-native/metro-config');
+const { getSentryExpoConfig } = require('@sentry/react-native/metro');
+
+// Your existing Sentry configuration (or any other configuration you have)
+const sentryConfig = getSentryExpoConfig(__dirname);
```

Comment:
Do you also use snetry? If not maybe better to remove the sentry part because not needed in fact

Reply:
I do not use sentry

Reply:
Should I leave template for user's config or just delete the user config part completely?